uefi-raw/src/protocol/device_path.rs

```diff
@@ -7,11 +7,13 @@ use crate::{guid, Guid};
 ///
 /// Note that the fields in this struct define the header at the start of each
 /// node; a device path is typically larger than these four bytes.
+#[derive(Debug)]
 #[repr(C)]
 pub struct DevicePathProtocol {
```

Comment:
Is there a reason why this is not a DST? I guess because handling DSTs is still complicated // requires the ptr_meta shim. Right?

Reply:
Sounds reasonable.